src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_create_with_auto_upgrade_channel.yaml

```diff
@@ -14,8 +14,8 @@ interactions:
       - --resource-group --name --location --enable-managed-identity --auto-upgrade-channel
         --ssh-key-value
       User-Agent:
```

Comment:
Your change in test_aks_commands.py is for nodepool snapshot, so you need to update   test_aks_nodepool_snapshot.yaml instead of test_aks_create_with_auto_upgrade_channel.yaml